### PR TITLE
[exporter/clickhouse] Fix ut when clickhouse-go/v2 version bump up

### LINKED ISSUE #17960: github.com/ClickHouse/clickhouse-go/v2 from 2.4.3 to 2.5.1 causes issues
This update appears to be causing issues:

```
=== RUN   TestLogsExporter_New/no_dsn
    exporter_logs_test.go:56: 
        	Error Trace:	/home/runner/work/opentelemetry-collector-contrib/opentelemetry-collector-contrib/exporter/clickhouseexporter/exporter_logs_test.go:56
        	            				/home/runner/work/opentelemetry-collector-contrib/opentelemetry-collector-contrib/exporter/clickhouseexporter/exporter_logs_test.go:80
        	Error:      	"create database:parse dsn address failed" does not contain "dial tcp: missing address"
        	Test:       	TestLogsExporter_New/no_dsn
--- FAIL: TestLogsExporter_New (0.00s)
    --- FAIL: TestLogsExporter_New/no_dsn (0.00s)
```

_Originally posted by @codeboten in https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/17950#pullrequestreview-1266238541_

See https://github.com/open-telemetry/opentelemetry-collector-contrib/actions/runs/3989607176/jobs/6842656560 for more details

### PR DESCRIPTION
**Pull request by Frapschen**
**Description:** Fix ut when clickhouse-go/v2 version bump up

resolve https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/17960

### CODE REVIEW

File: exporter/clickhouseexporter/exporter_logs_test.go [hunk 1]
```diff
@@ -63,7 +63,8 @@ func TestLogsExporter_New(t *testing.T) {
 	}{
 		"no dsn": {
 			config: withDefaultConfig(),
-			want:   failWithMsg("dial tcp: missing address"),
+			want:   failWithMsg("exec create logs table sql: parse dsn address failed"),
+			// exec create logs table sql: parse dsn address failed
```
Comment (dmitryax) [inline exporter/clickhouseexporter/exporter_logs_test.go:67]: redundant comment 
```suggestion
```

**Comment by dmitryax** [comment]
+skipchangelog

**Comment by Frapschen** [comment]
not relate fail https://github.com/open-telemetry/opentelemetry-collector-contrib/actions/runs/4089471343/jobs/7052232713#step:6:544
```
    integration_test.go:70: 
        	Error Trace:	/home/runner/work/opentelemetry-collector-contrib/opentelemetry-collector-contrib/receiver/mysqlreceiver/integration_test.go:70
        	Error:      	Received unexpected error:
        	            	resource "map[mysql.instance.endpoint:localhost:3306]": scope "otelcol/mysqlreceiver": number of metrics doesn't match expected: 21, actual: 20
        	Test:       	TestMySqlIntegration/Running_mysql_version_8.0
--- FAIL: TestMySqlIntegration (0.00s)
    --- FAIL: TestMySqlIntegration/Running_mysql_version_8.0 (31.22s)
FAIL
```

**Review by dmitryax — APPROVED** [review]

**Pull request merged**